THIRD-PARTY-NOTICES.TXT

```diff
@@ -69,3 +69,10 @@ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+
+License notice for AIS parser (in NMEA-0183 binding)
```

Comment:
We should check with @richlander, but I believe we have to also paste the whole license here as well.

Reply:
Yes.